ui/app/AppMain.qml

```diff
@@ -170,6 +174,12 @@ Item {
         }
     }
 
+    property Component changeProfilePicComponent: Component {
+        ChangeProfilePicModal {
+            profileStore: appMain.rootStore.profileSectionStore.profileStore
```

Comment:
This probably needs a destroy on close

Reply:
@jrainville it is destroyed. `ChangeProfilePicModal` component is "smart" and it doesn't require explicit `destroy()` call out of the component since it already has `onClosed` slot inside which calls `destroy()`. But this also means that this component should be used only if it's dynamically created.